frontend/src/components/LogIn/login.css

```diff
@@ -0,0 +1,120 @@
+body {
+    background-color: #F4EFFA;
+    padding-top: 10%;
+    padding-bottom: 10%;
+}
+
+h1 {
+    letter-spacing: 5px;
+    margin-bottom: 5%;
+}
+
+.loginGroup {
```

Comment:
by convention, css class names are not camel cased by snake cased. `login-group` small nit, but might be good to adopt the convention!